pkg/sentryclient/sentryclient.go

```diff
@@ -0,0 +1,48 @@
+package sentryclient
+
+import (
+	"os"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+	log "github.com/uswitch/heimdall/pkg/log"
+)
+
+var (
+	sentryActive = false
+)
+
+func SentryErr(exception error) {
+	if sentryActive {
+		sentry.CaptureException(exception)
+		log.Sugar.Debug("Error has been sent to Sentry!")
+	}
+}
+
+func SentryMessage(message string) {
+	if sentryActive {
+		sentry.CaptureMessage(message)
+		log.Sugar.Debug("Message has been sent to Sentry!")
+	}
+}
+
+// SetupSentry is the function setting up the Sentry access
+func SetupSentry() {
+	endpoint, exists := os.LookupEnv("sentry_endpoint")
+
+	if exists {
+		err := sentry.Init(sentry.ClientOptions{
+			Dsn: endpoint,
+		})
+		if err != nil {
+			log.Sugar.Fatalf("sentry.Init: %s", err)
+		}
+		// Flush buffered events before the program terminates.
+		defer sentry.Flush(2 * time.Second)
```

Comment:
This defer will be executed immediately when `SetupSentry()` ends. It's likely we want to instead return a function that wraps the Flush call so we can defer it in `main()`.